importer/importer.py

```diff
@@ -106,10 +107,10 @@ def create_connection(arguments):
                                          "data_files": {"counties": "estonia_counties.json"}},
                    "monuments_se-fornmin_(sv)":
                    {"class": SeFornminSv,
-                    "data_files":
-                    {"municipalities": "sweden_municipalities.json",
-                     "socken": "sweden_socken.json",
-                     "types": "se-fornmin_(sv)_types.json"}},
+                             "data_files": {"municipalities": "sweden_municipalities.json",
```

Comment:
Indentation looks wrong here. "data_files" and "class" should be on the same level.